ui/src/reducers/__tests__/index.test.js

```diff
@@ -1,38 +1,41 @@
 import reducer from '../index';
 import {
   APPS_SUCCESS,
-  APP_DETAILS_SUCCESS,
   APPS_SORT,
-  APP_DETAILS_SORT,
   APPS_FAILURE,
-  TOGGLE_HEADER_DROPDOWN
+  TOGGLE_HEADER_DROPDOWN,
+  APP_SUCCESS, APP_FAILURE,
+  APP_VERSIONS_SORT
 } from '../../actions/types.js';
 import { SortByDirection, sortable } from '@patternfly/react-table';
 
 describe('reducer', () => {
   const initialState = {
     apps: { rows: [], data: {} },
     sortBy: { direction: SortByDirection.asc, index: 0 },
-    appDetailsSortDirection: { direction: SortByDirection.asc, index: 0 },
+    appVersionsSortDirection: { direction: SortByDirection.asc, index: 0 },
     columns: [
-      { title: 'App Name', transforms: [ sortable ] },
-      { title: 'App ID', transforms: [ sortable ] },
-      { title: 'Deployed Versions', transforms: [ sortable ] },
-      { title: 'Current Installs', transforms: [ sortable ] },
-      { title: 'Launches', transforms: [ sortable ] }
+      { title: 'App Name', transforms: [sortable] },
+      { title: 'App ID', transforms: [sortable] },
+      { title: 'Deployed Versions', transforms: [sortable] },
+      { title: 'Current Installs', transforms: [sortable] },
+      { title: 'Launches', transforms: [sortable] }
     ],
-    appDetailRows: [],
-    appDetailColumns: [
-      { title: 'App Version', transforms: [ sortable ] },
-      { title: 'Current Installs', transforms: [ sortable ] },
-      { title: 'Launches', transforms: [ sortable ] },
-      { title: 'Last Launched', transforms: [ sortable ] },
-      { title: 'Disable on Startup', transforms: [ sortable ] },
-      { title: 'Custom Disable Message', transforms: [ sortable ] }
+    appVersionsColumns: [
+      { title: 'App Version', transforms: [sortable] },
+      { title: 'Current Installs', transforms: [sortable] },
+      { title: 'Launches', transforms: [sortable] },
+      { title: 'Last Launched', transforms: [sortable] },
+      { title: 'Disable on Startup', transforms: [sortable] },
+      { title: 'Custom Disable Message', transforms: [sortable] }
     ],
     isAppsRequestFailed: false,
     currentUser: 'currentUser',
-    isUserDropdownOpen: false
+    isUserDropdownOpen: false,
+    app: {
+      deployedVersions: { rows: [], data: {} }
```

Comment:
@craicoverflow there seems to already be a `deployedVersions` array inside the data object. Could we remove it from here so that the data object contains the following:


<img width="327" alt="Screenshot 2019-03-14 at 17 19 33" src="https://user-images.githubusercontent.com/6498727/54377641-720fb880-467d-11e9-9635-f253eaad7054.png">

As it stands it's nested as below:

<img width="368" alt="Screenshot 2019-03-14 at 17 20 08" src="https://user-images.githubusercontent.com/6498727/54377648-7936c680-467d-11e9-862f-6c179ee6321d.png">

Reply:
Marking as resolved. thanks for the update